src/pages/Hosts.tsx

```diff
@@ -92,9 +120,41 @@ const Hosts: React.FC = () => {
       <H2>Evergreen Hosts</H2>
       <ErrorBoundary>
         <TableControlOuterRow>
-          <div data-cy="filtered-hosts-count">
-            {hasFilters && `Showing ${filteredHostCount} of ${totalHostsCount}`}
-          </div>
+          <SubtitleDataWrapper>
+            <Disclaimer data-cy="filtered-hosts-count">
+              {`Showing ${
+                hasFilters ? filteredHostCount : totalHostsCount
+              } of ${totalHostsCount}`}
+            </Disclaimer>
+            {selectedHostIds.length > 0 && (
+              <HostsSelectionWrapper>
+                <Badge variant={Variant.Blue} data-cy="hosts-selection-badge">
+                  {selectedHostIds.length} Selected
+                </Badge>
+                <ButtonWrapper>
+                  <Button dataCy="update-status-button">Update Status</Button>
+                </ButtonWrapper>
+                <ButtonWrapper>
+                  <Popconfirm
+                    title={`Restart Jasper for ${selectedHostIds.length} host${
+                      selectedHostIds.length > 1 ? "s" : ""
```

Comment:
I think this should be === instead of >

Reply:
![Screen Shot 2020-08-07 at 3 06 39 PM](https://user-images.githubusercontent.com/15262143/89679704-9f3ab780-d8bf-11ea-80d0-756f0e961946.png)

why?